crane-cloud/backend
# Added scheduler for notification of credits expiration

# Description

Added scheduler to send notification of credits expiration 1 month before expiration

## Type of change

Please delete options that are not relevant.




## Trello Ticket ID

https://trello.com/c/EF3ZDPwQ/1177-email-notification-for-credits-expiration

## How Can This Be Tested?

Change the expiration check date for send_expiration_notification task (notification_datetime) in tasks/_init_.py to date of testing. Set expiration date one month after date of testing. Run celery. User should receive email notification.


## Checklist:
